.github/workflows/scripts/node-badge.mjs

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+//! Calculate the percentage of games from node that we support
+// Expects node-gamedig checkout out in git root /node-gamedig
+// Expects the generic example to output a list of game IDs when no arguments are provided
+
+import process from "node:process";
+import { closeSync, openSync, writeSync } from "node:fs";
+import { spawnSync } from "node:child_process";
+
+function setOutput(key, value) {
```

Comment:
Just as a petty comment, please use arrow functions.

Reply:
As in?
```js
const setOutput = (key, value) => {
  /* ... */
}
```

Reply:
Yes